### ISSUE TITLE
Confirm dataset used

### ISSUE THREAD

**Issue by robmarkcole** [ClarkCGA/multi-temporal-crop-classification-baseline#6]
Hi 
the `default_config` references a dataset:
```
train_dataset_name: chips_filtered_13_classes_complete
```
Can you confirm this is the [exact dataset from HF](https://huggingface.co/datasets/ibm-nasa-geospatial/multi-temporal-crop-classification), and not a modified version (implied by the reference to filtering)
Thanks

**Comment by robmarkcole** [comment]
Using the official dataset I have `len(train_dataset) = 3083`

**Comment by samKhallaghi** [comment]
Hi Robin,
This is the exact dataset. "chips_filtered_13_classes_complete" was our internal name for the dataset. Please update the config with the new name.

**Comment by irishdevops** [comment]
@robmarkcole  If you would like to get the filtered df, you can map the original df to their labels and then just select the ones  that appear mentioned in this project repo

**Comment by irishdevops** [comment]
@robmarkcole If you would like to get the filtered df, you can map the original df to their labels.Then filter by the ones that appear mentioned in this project repo
